src/main/java/com/northconcepts/datapipeline/UpsertWithJdbcMultiUpsertWriter.java

```diff
@@ -0,0 +1,61 @@
+package com.northconcepts.datapipeline;
+
+import com.mysql.cj.jdbc.MysqlDataSource;
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.csv.CSVReader;
+import com.northconcepts.datapipeline.internal.jdbc.JdbcFacade;
+import com.northconcepts.datapipeline.jdbc.JdbcConnectionFactory;
+import com.northconcepts.datapipeline.jdbc.JdbcMultiUpsertWriter;
+import com.northconcepts.datapipeline.jdbc.upsert.MySqlUpsert;
+import com.northconcepts.datapipeline.job.Job;
+
+import javax.sql.DataSource;
+import java.io.File;
+
+public class UpsertWithJdbcMultiUpsertWriter {
```

Comment:
Let's call this (and the content) `UpsertWithMultipleJdbcConnections` since users might search for something with "multiple connections" in it versus our name for it.  

See a short video on the topic: https://youtu.be/Lhseljh_mnw

Reply:
A very nice video. Thanks for sharing.